README-dnssec.md

```diff
@@ -0,0 +1,296 @@
+# DNSSEC setup for dnsseed
+
+##### Table of Contents  
+[Requirements](#Requirements)  
+[Software needed](#Software)  
+[Setup](#Setup)  
+[Testing](#Testing)  
+[Example](#Example)  
+[Improvements](#Improvements)  
+[Links](#Links)  
+
+
+<a name="Requirements"/>
+
+## Requirements
+* run only this service on the host (security precaution)
+
+<a name="Software"/>
+
+## Software needed
+
+* tor (if you like to seed tor nodes)
+* bind9
+* dig (debian package bind9-dnsutils)
+* nsupdate (debian package bind9-dnsutils)
+* bitcoin-seeder
+* ufw
+* bash
```

Comment:
I don't disagree with requiring bash, but it seems like it isn't strictly needed right now. If you'd require bash, you may as well use its fancy features like `set -euxo pipefail` as I have commented in that script

Reply:
I prefer to run scripts with a selected shell not wirh ```sh```

I will also add ```set -euo pipefail``` in the script.